src/components/Modal/ProfileModal.tsx

```diff
@@ -8,8 +8,38 @@ export class ProfileModal extends React.Component<{
   close: Function;
   user: firebase.User;
   userImage: string | null;
+  isSubscriber?: boolean;
+  discordUsername?: string;
+  discordDiscriminator?: string;
 }> {
-  public state = { resetDisabled: false, deleteConfirmOpen: false };
+  public state = {
+    resetDisabled: false,
+    deleteConfirmOpen: false,
+  };
+
+  authDiscord = () => {
+    const url = process.env.REACT_APP_DISCORD_AUTH_URL;
```

Comment:
is this different/unique for each app?

Reply:
Yes, you will have to generate this as the app/bot owner.

Reply:
It contains the discord app client ID and redirect URL.